libs/tree-sitter-wing/grammar.js

```diff
@@ -223,6 +224,16 @@ module.exports = grammar({
         field("block", $.block),
       ),
 
+    try_catch_statement: ($) =>
+      seq(
+        "try",
+        field("block", $.block),
+        "catch",
+        optional(field("exception_identifier", $.identifier)),
```

Comment:
technically the `catch` block is optional (you should be able to do `try/finally`)

Reply:
If we add that, we need to also update the spec: https://docs.winglang.io/reference/spec#19-error-handling

Reply:
updated code and spec.